src/thunderbots/software/util/parameter/dynamic_parameters.cpp

```diff
@@ -71,4 +71,29 @@ namespace Util::DynamicParameters
         Parameter<double> angular_sensitivity("xbox_demo_angular_sensitivity", 1.0);
     }  // namespace XBoxControllerDemo
 
+    namespace NetworkInput
+    {
+        namespace Filter
+        {
+            namespace PartileFilter
+            {
+                Parameter<int32_t> PARTICLE_FILTER_NUM_CONDENSATIONS(
```

Comment:
We should remove the `PARTICLE_FILTER_` prefix from all of these, as they're already in the `NetworkInput::Filter::ParticleFilter` namespace.